src/style/sections/layers/face.less

```diff
@@ -215,6 +215,7 @@ input[type="text"].face__name {
     margin-left: 0;
     width: @properties-width - 4rem;
     border-radius: @face-border-radius;
+    z-index: 100;
```

Comment:
This seemed to fix #1671. Obviously, 100 may not be the right value here. Any suggestions?